paperwork_backend/docsearch.py

```diff
@@ -252,7 +252,7 @@ class DocSearch(object):
     LABEL_STEP_UPDATING = "label updating"
     LABEL_STEP_DESTROYING = "label deletion"
 
-    def __init__(self, rootdir, indexdir=None, language=None,
+    def __init__(self, rootdir, indexdir=None, index_in_workdir=None, language=None,
```

Comment:
Can you add this argument at the end please ? So it doesn't break the Flatpak build if it is built before the corresponding changes in the frontend are merged.